lib/new_relic/tracer/macro.ex

```diff
@@ -17,6 +17,7 @@ defmodule NewRelic.Tracer.Macro do
 
   # Take no action on the definition of a function head
   def __on_definition__(_env, _access, _name, _args, _guards, nil), do: nil
+  def __on_definition__(_env, _access, _name, _args, _guards, []), do: nil
```

Comment:
Nice, so was it a function head definition that was causing the error? ie:

```elixir
def foo(bar \\ :default)
```

or was this some other case..

Reply:
Yep, it was a function head definition.

Example of one from our codebase:
``` elixir
defp parse_response(http_result, ok_code_or_codes \\ 200)
```